policyengine_us/variables/gov/states/id/tax/income/credits/id_aged_or_disabled_credit.py

```diff
@@ -0,0 +1,38 @@
+from policyengine_us.model_api import *
+
+
+class id_aged_or_disabled_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho aged or disabled credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        age = person("age", period)
+        head_or_spouse = person("is_tax_unit_head_or_spouse", period)
+        disabled = person("is_disabled", period)
+        p = parameters(
+            period
+        ).gov.states.id.tax.income.credits.aged_or_disabled
+        age_eligible = (age >= p.age_eligibility) & ~head_or_spouse
+        # To claim aged or disabled credit, filers also have to maintain a household for family members
+        # and provide more than one-half of the family member’s support for the year
+        care_and_support_payment = person("care_and_support_payment", period)
+        care_and_support_costs = person("care_and_support_costs", period)
+        support_payment_ratio = np.zeros_like(care_and_support_costs)
+        mask = care_and_support_costs != 0
+        support_payment_ratio[mask] = (
+            care_and_support_payment[mask] / care_and_support_costs[mask]
+        )
+        payment_eligible = support_payment_ratio > p.cost_rate
+        eligible_person = (age_eligible | disabled) & payment_eligible
+        total_eligible_people = sum(eligible_person)
+        capped_eligible_people = min_(total_eligible_people, p.max_amount)
+        # The aged or disabled credit can only be claimed if the aged
+        # or disabled deduction is not claimed
+        aged_deduction = tax_unit("id_aged_or_disabled_deduction", period)
```

Comment:
don't we want them to choose the option that minimizes tax liability? this will require branching

Reply:
These passages of the tax form led us to believe that we first check for the availability of the deduction and if it is0, then we can claim the credit, 

Let us know if misinterpreted

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/b0dbefaa-3b20-43ff-a8fd-4c41a1156039)
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/e5458959-0e8a-4164-a612-597016f5ca77)

Reply:
the screenshots are from the deduction explanation as well as credit, respectively

Reply:
Why wouldn't they claim the deduction if they can claim the credit? I thought it was the same criteria.

In any case, I don't think that language precludes the option for a filer to forgo the deduction to take the credit instead, so I do think we need to branch.

Reply:
I think under current law they'd always take the credit over the deduction, given the tax rate is 5.8%, so the deduction should generally be only worth $58 vs $100 for the credit. So we could add a variable for now that has them select which, and have that default to the credit for now, with an issue to branch in the future if that's easier.

Reply:
@PavelMakarchuk you re-requested my review but this remains unresolved afaict

Reply:
@Jingj-WANG please create a variable which will allows to select the larger of the credit or the deduction and file an issue to branch eventually

Reply:
add a variable called "id_selected_aged_or_disabled_credit"

Reply:
This will need a formula, and that will require branching. If you're doing it outside this pr please file an issue.

Reply:
#3380 will be the branching issue